src/style-auro-calendar.scss

```diff
@@ -148,16 +148,20 @@
   left: 50%;
 
   display: none;
-  width: var(--calendar-width); // match the calendar month width; This needs to be deal with when doing the tiny mobile device layout.
+  width: 100%; // match the calendar month width; This needs to be deal with when doing the tiny mobile device layout.
```

Comment:
Please review the Figma spec. 

You have this:
<img width="451" alt="Screen Shot 2023-11-22 at 6 16 22 PM" src="https://github.com/AlaskaAirlines/auro-datepicker/assets/181089/5b9c2e65-e295-4efe-bdfa-d18a486e6c7f">

The design spec is looking for this:
<img width="511" alt="Screen Shot 2023-11-22 at 6 17 18 PM" src="https://github.com/AlaskaAirlines/auro-datepicker/assets/181089/a3e7ef06-33e7-4b7e-a1b4-85f90c51ea37">